app/views/layouts/mailer.html.erb

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
Looks like this will be a great place to start if/when we add in the information to allow for the app to send out emails.